roles/logrotate/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Install base package
+  apt:
+    name: logrotate
+    state: installed
+
+- name: Install config
+  template:
+    src: etc/logrotate.d/template.conf
+    dest: /etc/logrotate.d/{{ item.name }}
+  with_items: "{{ logrotate_configs }}"
+  when: logrotate_configs|length > 0
```

Comment:
That's a correct nit. If `logrotate_configs` is an empty list then the task won't fire.